Dockerfile

```diff
@@ -21,7 +21,6 @@ RUN apk --no-cache add \
     postgresql-libs \
     tzdata \
     nodejs \
-    yarn \
```

Comment:
Why is this getting dropped?

Reply:
Related to the previous comment, no need for it anymore. :)